src/python/grpcio_tests/tests/stress/client.py

```diff
@@ -90,6 +101,18 @@ def _parse_weighted_test_cases(test_case_args):
     weighted_test_cases[test_case] = int(weight)
   return weighted_test_cases
 
+def get_channel(target, args):
```

Comment:
The lack of a leading underscore on this function's name indicates that you expect it to be used outside this module and support such usage - I doubt that that is as you intended but if it is add a doc string to the function.